templates/registration/includes/profile_music.html

```diff
@@ -1,12 +1,34 @@
 <div class="investments">
-    <h2>MY MUSIC</h2>
+    <h2>MUSIC</h2>
     <div class="break">
         <hr>
     </div>
     <div class="row small-up-1 medium-up-2 large-up-4 centered artistlist">
         {% for album in albums %}
             <div class="column">
-                {% include "object_preview.html" with object=album img=album.artwork.img %}
+                <div class="artist-list-image img-list">
+                    {% include "object_preview.html" with object=album img=album.artwork.img %}
+                </div>
+                <div>
+                    <p>
+                        {% for disc in album.discs %}
+                            {% for track in disc %}
+                                {% if track.total_streams %}
```

Comment:
Total all of the streams / downloads from each track on the album. 

When we get the reports back from distribution, they are all reported by individual songs - not the album as a whole. 

So it would look like, [sum all streams from album songs] [sum all downloads from album songs]